config/maven/updateRules.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we rename the file to be `versions-plugin-rules.xml`? I think this is a bit more descriptive.

Reply:
renamed, thanks